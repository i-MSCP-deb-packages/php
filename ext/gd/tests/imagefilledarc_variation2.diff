001+ cd10da83a96d9f639ed83379a814e0ea
001- b8b572812b3c85678f6c38c4ecca7619