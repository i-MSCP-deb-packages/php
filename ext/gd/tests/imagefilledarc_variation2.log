
---- EXPECTED OUTPUT
b8b572812b3c85678f6c38c4ecca7619
---- ACTUAL OUTPUT
cd10da83a96d9f639ed83379a814e0ea
---- FAILED
