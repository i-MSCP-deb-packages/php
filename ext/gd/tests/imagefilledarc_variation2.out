cd10da83a96d9f639ed83379a814e0ea